release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -77,6 +77,11 @@ Because {product-title} {product-version} now uses a {op-system-base} 9.2 based
 
 User-defined tags for Microsoft Azure was previously introduced as Technology Preview in {product-title} 4.13 and is now generally available in {product-title} 4.14. For more information, see xref:../installing/installing_azure/installing-azure-customizations.adoc#installing-azure-user-defined-tags_installing-azure-customizations[Configuring the user-defined tags for Azure].
 
+[id="ocp-4-14-user-defined-tags-gcp"]
+==== User-defined labels and tags for Google Cloud Platform (Technology Preview)
+
+You can now configure user-defined labels and tags in Google Cloud Platform (GCP) for grouping resources and for managing resource access and cost. User-defined labels can be applied only to resources created by {product-title} installation program and its core components. User-defined tags can be applied only to resources created by the {product-title} Image Registry Operator. For more information, see xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installing-gcp-user-defined-labels-and-tags_installing-gcp-customizations[Managing the user-defined labels and tags for GCP].
```

Comment:
```suggestion
You can now configure user-defined labels and tags in Google Cloud Platform (GCP) for grouping resources and for managing resource access and cost. User-defined labels can only be applied to resources you create using the installation program and its core components. User-defined tags can only be applied to resources you create using {product-title} Image Registry Operator. For more information, see xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installing-gcp-user-defined-labels-and-tags_installing-gcp-customizations[Managing the user-defined labels and tags for GCP].
```

Reply:
https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism

Reply:
The suggestion above indicates that it can "only be applied (but no other action can be performed)" whereas the message to be conveyed is that it is applied ONLY to particular resources as stated. 

Place the word only immediately before the word or phrase that it modifies.
Mine follows the latter
![image](https://github.com/openshift/openshift-docs/assets/84562993/0c8f5070-9529-40f6-9d6b-f76025bc1600)

Reply:
Yeah, you're correct. Sorry got twisted up in my suggestion.